src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectVotingController.kt

```diff
@@ -0,0 +1,114 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse403
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/projects/{projectId}/votes")
+@RestController
+class ProjectVotingController() {
+  @ApiResponse200
+  @ApiResponse403
+  @ApiResponse404
+  @GetMapping
+  @Operation(
+      summary = "Gets vote selections for a single project.",
+      description =
+          "List every vote selection for this project, organized by phases. Each phase will " +
+              "contain a list of eligible voters and their selections. If a `voteOption` is " +
+              "`null`, the voter has not yet casted a vote. ")
```

Comment:
Might make sense to put that last piece of information in a schema annotation on `voteOption` itself (either in addition to having it here or instead of).